packages/app/src/vis-packs/nexus/utils.ts

```diff
@@ -41,8 +41,9 @@ function assertNxDataGroup(group: Group): void {
 
 export function isNxGroup(entity: Entity): boolean {
   return (
-    isGroup(entity) &&
-    (isNxDataGroup(entity) || !!getAttributeValue(entity, 'default'))
+    (isGroup(entity) &&
+      (isNxDataGroup(entity) || !!getAttributeValue(entity, 'default'))) ||
+    !!getAttributeValue(entity, 'NX_class')
```

Comment:
There's a parenthesis issue, I think.

Reply:
Thanks, I added a simple `hasAttribute` wrapper to lighten the syntax a bit

Reply:
Ah yeah, good call